### PR TITLE
[IMP]change the api to the latest v2.0.

### PR DESCRIPTION
**Pull request by Rona111**
this is the version for V8

### CODE REVIEW

File: website_baidu_map/website_baidu_map.xml [hunk 1]
```diff
@@ -10,19 +10,21 @@
                     #allmap{width:100%;height:250px;}
                     p{margin-left:5px; font-size:14px;}
                 </style>
-                <script type="text/javascript" src="http://api.map.baidu.com/api?v=1.3"></script>
+                <script type="text/javascript" src="http://api.map.baidu.com/api?v=2.0&amp;ak=zFXL4R2eaZjkUSxsGDCAGLN8"></script>
```
Comment (elicoidal) [inline website_baidu_map/website_baidu_map.xml]: Is there a way to have a parameter in properties and chose http or https?

**Comment by Rona111** [comment]
cc @elicoidal @duanyp1991 @siyuancn

File: website_event_baidu_map/website_event_baidu_map.xml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <openerp>
     <data>
-        <template id="company_description_inherited" name="map" inherit_id="website.company_description" customize_show="True">
-            <xpath expr="a" position="replace">
+        <template id="event_description_full_inherited" name="map" inherit_id="website_event.event_description_full">
```
Comment (siyuancn) [inline website_event_baidu_map/website_event_baidu_map.xml:4]: why change to website_event?

File: website_event_baidu_map/README.rst [hunk 3]
```diff
@@ -2,18 +2,28 @@
...
+ * have basic modules installed (website_event)
+
+
+Configuration
+
+=============
+
+To configure this module, you need to:
+
+* Go to Settings/Parameters/System Parameters and create a record which Key must be named baidu.api and the Value like [http://api.map.baidu.com/api]?v=[2.0]&ak=[your ak].
```
Comment (siyuancn) [inline website_event_baidu_map/README.rst:25]: Please add the description for how to get the ak in baidu

Comment (siyuancn) [inline website_event_baidu_map/README.rst:25]: Please test your code in https environment.

File: website_event_baidu_map/website_event_baidu_map.xml [hunk 4]
```diff
@@ -10,17 +10,18 @@
...
                 <div id="allmap"></div>
                 <script type="text/javascript">
-                    // 百度地图API功能
-                    var map = new BMap.Map("allmap");          
+                    // BaiDu map API v2.0 function
+                    var map = new BMap.Map("allmap"); 
+                    map.addControl(new BMap.MapTypeControl());    
                     map.centerAndZoom(new BMap.Point(121.480237, 31.236305), 11);
-                    map.enableScrollWheelZoom();
+                    map.enableScrollWheelZoom(true);
```
Comment (siyuancn) [inline website_event_baidu_map/website_event_baidu_map.xml:20]: what's the usage of the parameter true?

**Comment by noahzaozao** [comment]
https://github.com/Elico-Corp/odoo-addons/pull/22

**Pull request merged**